docs/topics/testing/tools.txt

```diff
@@ -1605,6 +1605,45 @@ your test suite.
             Person.objects.create(name="Aaron")
             Person.objects.create(name="Daniel")
 
+.. _topics-tagging-tests:
+
+Tagging tests
+-------------
+
+.. versionadded:: 1.9
+
+You can tag your tests to group them- for example fast and slow tests::
+
+    from django.test.utils import tag
+
+    class SampleTestCase(TestCase):
```

Comment:
IIUC the implementation also supports tagging entire test cases (if it doesn't, I believe it should -- this will often be an important shortcut IMO). In that case it seems like the documentation should demonstrate such usage, too.